src/app/CommandHandler.cpp

```diff
@@ -121,7 +121,7 @@ CHIP_ERROR CommandHandler::AddStatusCode(const CommandPathParams * apCommandPath
     CHIP_ERROR err = CHIP_NO_ERROR;
     StatusElement::Builder statusElementBuilder;
 
-    err = PrepareCommand(apCommandPathParams);
+    err = PrepareCommand(apCommandPathParams, true);
```

Comment:
use `PrepareCommand(apCommandPathParans, true /* isStatus */)` to explain what the boolean means.  same below.

PrepareCommand bool argument is not obvious and for finish  I would personally expect success/failure if I ever saw a bool argument. Naked bools are unclear.

alternative would be to add constants like `constexpr bool kIsStatusCommand = true` and then call things like:

`PrepareCommand(apCommandPathParams, kIsStatusCommand)` or maybe even better use an enum of `kRegularCommand, kStatusCommand` and pass in those to be typesafe.

Reply:
Done, thanks